images/web/start.sh

```diff
@@ -39,7 +42,7 @@ while "$flag" = true; do
     sleep 2
   done &
 
-  bundle exec rake assets:precompile --trace
+  # bundle exec rake assets:precompile --trace
```

Comment:
I thought we needed the precompile but I guess may be also not needed at this time.